features/step_definitions/basic_steps.rb

```diff
@@ -22,6 +22,10 @@
   expect(current_path).to eq root_path
 end
 
+Then("show me the page") do
+  save_and_open_page
+end
+
```

Comment:
Remove this before we can merge. We don't want this in our deployed code. :)

Reply:
Ok